src/components/Text/index.js

```diff
@@ -51,7 +52,7 @@ const styles = StyleSheet.create({
 Text.propTypes = {
   style: PropTypes.object,
   colorVariant: PropTypes.oneOf(['primary', 'secondary', 'black', 'white', 'gray', 'gray2', 'gray3']),
-  fontVariant: PropTypes.oneOf(['h1', 'h2', 'h3', 'description', 'body', 'date'])
+  fontVariant: PropTypes.oneOf(['h1', 'h2', 'h3', 'title', 'description', 'body', 'date'])
 }
```

Comment:
Be careful, this component is growing too much. Maybe what we need here are different components that _extend_ `Text` (by composition). Each new component would only apply its specific styles.